extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -213,7 +213,7 @@ export const SendAmount = ({
     if (values.destinationAsset) {
       dispatch(saveDestinationAsset(values.destinationAsset));
       isDestAssetScam = isAssetSuspicious(
-        accountBalances.balances?.[destinationAsset].blockaidData,
+        accountBalances.balances?.[destinationAsset]?.blockaidData,
```

Comment:
this was throwing an error for soroswap assets because the user doesn't actually have these assets in their balances yet